res/database/Default/user/therob.xml

```diff
@@ -2312,19 +2613,23 @@
 			<title>
 				<de>%GROANERS% %TYPE%</de>
 				<en>%GROANERS% %TYPE%</en>
+				<pl>%GROANERS% na scenie - %TYPE%</pl>
 			</title>
 			<description>
 				<de>Ein Moderator lässt seine Gäste Sketche aufführen und gibt ihnen spontane Regieanweisungen.</de>
 				<en>A host has his guests perform skits and gives them spontaneous stage directions.</en>
+				<pl>Gospodarz zleca swoim gościom odgrywanie skeczy i udziela im spontanicznych wskazówek scenicznych.</pl>
 			</description>
 			<variables>
 				<type>
 					<de>am laufenden Band|ohne Ende|auf Kommando|im Dutzend|par excellence|ganz spontan</de>
 					<en>churned out|unlimited|on Cue|by the Dozen|par excellence|all spontaneous</en>
+					<pl>non-stop|bez ograniczeń|bez końca|dla mas|do doskonałości|wszystko spontanicznie</pl>
 				</type>
 				<groaners>
 					<de>Gags|Kalauer|Witze|Scherze|Ulk|Zoten|Späßchen</de>
 					<en>Gags|Laughs|Jests|Jokes|Fun|Pranks|Banter</en>
+					<pl>Gagi|Śmieszki|Żarty|Zabawy|Skecze|Pokazy</pl>
```

Comment:
different number of alternative will cause problems

Reply:
I don't know why I overlooked. corrected.